src/EventService.Business/Commands/Event/CreateEventCommand.cs

```diff
@@ -102,6 +110,16 @@ private async Task SendInviteEmailsAsync(List<Guid> userIds, string eventName)
         validationResult.Errors.ConvertAll(er => er.ErrorMessage));
     }
 
+    List<DbCategory> dbCategories = new();
+
+    if (!request.CategoriesRequests.IsNullOrEmpty())
+    {
+      foreach (CreateCategoryRequest categoryRequest in request.CategoriesRequests)

```

Comment:
May be like this? 
![image](https://github.com/LT-Students/DigitalOffice-EventService/assets/43515136/3445add8-d5e5-463d-a05f-b5b88772d415)